<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>509</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Robot Monitoring</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>User Command</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textEdit_3">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Communication</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textEdit_2">
              <property name="textInteractionFlags">
               <set>Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QOpenGLWidget" name="openGLWidget"/>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string>Linear Speed</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="lcdNumber">
                <property name="maximumSize">
                 <size>
                  <width>128</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Filled</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="text">
                 <string>Angle</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="lcdNumber_2">
                <property name="maximumSize">
                 <size>
                  <width>128</width>
                  <height>24</height>
                 </size>
                </property>
                <property name="smallDecimalPoint">
                 <bool>false</bool>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Filled</enum>
                </property>
                <property name="value" stdset="0">
                 <double>267.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Position X</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLCDNumber" name="lcdNumber_3">
                  <property name="maximumSize">
                   <size>
                    <width>128</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::Box</enum>
                  </property>
                  <property name="segmentStyle">
                   <enum>QLCDNumber::Filled</enum>
                  </property>
                  <property name="value" stdset="0">
                   <double>-12.449999999999999</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Position Y</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLCDNumber" name="lcdNumber_4">
                  <property name="maximumSize">
                   <size>
                    <width>128</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="segmentStyle">
                   <enum>QLCDNumber::Filled</enum>
                  </property>
                  <property name="value" stdset="0">
                   <double>123.780000000000001</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushRun">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>X (cm)</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox">
                <property name="maximumSize">
                 <size>
                  <width>64</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Y (cm)</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
                <property name="maximumSize">
                 <size>
                  <width>64</width>
                  <height>24</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>256</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushStop">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>72</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>72</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menusetup">
    <property name="title">
     <string>Setup</string>
    </property>
    <widget class="QMenu" name="menuPort_Com">
     <property name="title">
      <string>Port Com</string>
     </property>
     <addaction name="actionRefresh"/>
     <addaction name="separator"/>
     <addaction name="actionNo_Port"/>
     <addaction name="actionCOM_1"/>
     <addaction name="actionCOM_2"/>
     <addaction name="actionCOM_3"/>
     <addaction name="actionCOM_4"/>
     <addaction name="actionCOM_5"/>
     <addaction name="actionCOM_6"/>
     <addaction name="actionCOM_7"/>
     <addaction name="actionCOM_8"/>
     <addaction name="actionCOM_9"/>
     <addaction name="actionCOM_10"/>
     <addaction name="actionCOM_11"/>
     <addaction name="actionCOM_12"/>
     <addaction name="actionCOM_13"/>
     <addaction name="actionCOM_14"/>
     <addaction name="actionCOM_15"/>
     <addaction name="actionCOM_16"/>
    </widget>
    <widget class="QMenu" name="menuBaudrate">
     <property name="title">
      <string>Baudrate</string>
     </property>
     <addaction name="action1200"/>
     <addaction name="action2400"/>
     <addaction name="action4800"/>
     <addaction name="action9600"/>
     <addaction name="action19200"/>
     <addaction name="action38400"/>
     <addaction name="action57600"/>
     <addaction name="action115200"/>
    </widget>
    <widget class="QMenu" name="menuData_Bits">
     <property name="title">
      <string>Data Bits</string>
     </property>
     <addaction name="action5"/>
     <addaction name="action6"/>
     <addaction name="action7"/>
     <addaction name="action8"/>
    </widget>
    <widget class="QMenu" name="menuParity">
     <property name="title">
      <string>Parity</string>
     </property>
     <addaction name="actionnone"/>
     <addaction name="actioneven"/>
     <addaction name="actionodd"/>
     <addaction name="actionspace"/>
     <addaction name="actionmark"/>
    </widget>
    <widget class="QMenu" name="menuStop_Bits">
     <property name="title">
      <string>Stop Bits</string>
     </property>
     <addaction name="action1_bits"/>
     <addaction name="action1_5_bits"/>
     <addaction name="action2_bits"/>
    </widget>
    <addaction name="menuPort_Com"/>
    <addaction name="menuBaudrate"/>
    <addaction name="menuData_Bits"/>
    <addaction name="menuParity"/>
    <addaction name="menuStop_Bits"/>
    <addaction name="separator"/>
    <addaction name="actionConfiguration"/>
    <addaction name="actionTest"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionSensor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menusetup"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Show</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="action8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action1200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1200</string>
   </property>
  </action>
  <action name="action2400">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2400</string>
   </property>
  </action>
  <action name="action4800">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4800</string>
   </property>
  </action>
  <action name="action9600">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>9600</string>
   </property>
  </action>
  <action name="action19200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>19200</string>
   </property>
  </action>
  <action name="action38400">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>38400</string>
   </property>
  </action>
  <action name="action57600">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>57600</string>
   </property>
  </action>
  <action name="action115200">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>115200</string>
   </property>
  </action>
  <action name="actioneven">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>even</string>
   </property>
  </action>
  <action name="actionodd">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>odd</string>
   </property>
  </action>
  <action name="actionmark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>mark</string>
   </property>
  </action>
  <action name="actionspace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>space</string>
   </property>
  </action>
  <action name="action1_bits">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1 bits</string>
   </property>
  </action>
  <action name="action1_5_bits">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1.5 bits</string>
   </property>
  </action>
  <action name="action2_bits">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2 bits</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionnone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>none</string>
   </property>
  </action>
  <action name="action5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="actionCOM_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 1</string>
   </property>
  </action>
  <action name="actionCOM_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 2</string>
   </property>
  </action>
  <action name="actionCOM_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 3</string>
   </property>
  </action>
  <action name="actionCOM_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 4</string>
   </property>
  </action>
  <action name="actionCOM_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 5</string>
   </property>
  </action>
  <action name="actionCOM_6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 6</string>
   </property>
  </action>
  <action name="actionCOM_7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 7</string>
   </property>
  </action>
  <action name="actionCOM_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 8</string>
   </property>
  </action>
  <action name="actionCOM_9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 9</string>
   </property>
  </action>
  <action name="actionCOM_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 10</string>
   </property>
  </action>
  <action name="actionCOM_11">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 11</string>
   </property>
  </action>
  <action name="actionCOM_12">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 12</string>
   </property>
  </action>
  <action name="actionCOM_13">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 13</string>
   </property>
  </action>
  <action name="actionCOM_14">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 14</string>
   </property>
  </action>
  <action name="actionCOM_15">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 15</string>
   </property>
  </action>
  <action name="actionCOM_16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM 16</string>
   </property>
  </action>
  <action name="actionNo_Port">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>No Port</string>
   </property>
  </action>
  <action name="actionSensor">
   <property name="text">
    <string>Sensor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
